Superposition/Tasks.qs

```diff
@@ -50,21 +50,8 @@ namespace Quantum.Kata.Superposition {
         // ...
     }
 
-
-    // Task 3*. Unequal superposition
-    // Inputs:
-    //      1) a qubit in the |0⟩ state.
-    //      2) angle alpha, in radians, represented as Double.
-    // Goal: prepare a cos(alpha) * |0⟩ + sin(alpha) * |1⟩ state on this qubit.
-    operation UnequalSuperposition (q : Qubit, alpha : Double) : Unit {
-        // Hint: Experiment with rotation gates from the Microsoft.Quantum.Intrinsic namespace.
-        // Note that all rotation operators rotate the state by _half_ of its angle argument.
-
-        // ...
-    }
-
-
-    // Task 4. Superposition of all basis vectors on two qubits
+    
```

Comment:
GitHub doesn't allow me to leave the comment where it belongs: let's define part I above the 1st task to match part II later in the kata, as part of splitting the tasks in two groups. (Similar format to Measurements kata as well)